MIGRATION.md

```diff
@@ -0,0 +1,37 @@
+# dp-frontend-renderer migration
+  
+====================
+
+The `dp-frontend-renderer` is being depricated and the files migrated to the relevant controller/service.
+
+The services are likely to be in a transient state, where a file has been migrated but the effected service has not been released to production. At this point, the files will still be in the renderer but pending removal.  
```

Comment:
effected => affected

maybe make it explicit that it's `dp-frontend-renderer` instead "renderer" just for clarity?